README-sycl.md

```diff
@@ -293,7 +317,9 @@ Note:
 
 Like:
 ```
-Using device **0** (Intel(R) Arc(TM) A770 Graphics) as main device
+detect 1 SYCL GPUs: [0] with top Max compute units:512
```

Comment:
I think the title of this 5th section can be re-write:

5. Verify the device ID in output

Verify to see if the selected GPU is shown in the output:

```
detect 1 SYCL GPUs: [0] with top Max compute units:512
```

Or

```
use 1 SYCL GPUs: [0] with Max compute units:512
```

Reply:
Yes, update